enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/consensus/LeaderInfo.java

```diff
@@ -32,11 +32,23 @@
 
     private final MemberId memberId;
     private final long term;
+    private boolean isStepDown;
 
     public LeaderInfo( MemberId memberId, long term )
     {
         this.memberId = memberId;
         this.term = term;
+        this.isStepDown = false;
+    }
+
+    public void stepDown()
```

Comment:
Is calling this method an instruction to the leader to step down, or a record of a leader that is stepping down? The language sounds like the former. Maybe `steppingDown` would be a better name, and more consistent with names elsewhere?